Description: Append .stable to libs/binaries
 This is to allow installing both main/alpha releases at once.
Author: Jason Rhinelander <jason@imaginary.ca>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-11-23

--- graft-1.5.1.orig/external/db_drivers/liblmdb/CMakeLists.txt
+++ graft-1.5.1/external/db_drivers/liblmdb/CMakeLists.txt
@@ -58,5 +58,5 @@ set_property(TARGET lmdb APPEND PROPERTY
 
 set_target_properties(lmdb PROPERTIES VERSION 0)
 install(TARGETS lmdb
-    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft
-    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft)
+    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable
+    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable)
--- graft-1.5.1.orig/external/easylogging++/CMakeLists.txt
+++ graft-1.5.1/external/easylogging++/CMakeLists.txt
@@ -58,6 +58,6 @@ set_property(TARGET easylogging APPEND P
 
 set_target_properties(easylogging PROPERTIES VERSION 0)
 install(TARGETS easylogging
-    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft
-    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft)
+    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable
+    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable)
 
--- graft-1.5.1.orig/src/CMakeLists.txt
+++ graft-1.5.1/src/CMakeLists.txt
@@ -98,8 +98,8 @@ function (monero_add_library name)
   set_property(TARGET "${name}" APPEND PROPERTY COMPILE_FLAGS "-fPIC")
   set_target_properties("${name}" PROPERTIES VERSION 0)
   install(TARGETS "${name}"
-    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft
-    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft)
+    ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable
+    LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE}/graft.stable)
 endfunction ()
 
 add_subdirectory(common)
--- graft-1.5.1.orig/src/blockchain_utilities/CMakeLists.txt
+++ graft-1.5.1/src/blockchain_utilities/CMakeLists.txt
@@ -93,7 +93,7 @@ add_dependencies(blockchain_import
 	version)
 set_property(TARGET blockchain_import
 	PROPERTY
-	OUTPUT_NAME "graft-blockchain-import")
+	OUTPUT_NAME "graft-blockchain-import.stable")
 install(TARGETS blockchain_import DESTINATION bin)
 
 monero_add_executable(blockchain_export
@@ -116,6 +116,6 @@ add_dependencies(blockchain_export
 	version)
 set_property(TARGET blockchain_export
 	PROPERTY
-	OUTPUT_NAME "graft-blockchain-export")
+	OUTPUT_NAME "graft-blockchain-export.stable")
 install(TARGETS blockchain_export DESTINATION bin)
 
--- graft-1.5.1.orig/src/daemon/CMakeLists.txt
+++ graft-1.5.1/src/daemon/CMakeLists.txt
@@ -102,5 +102,5 @@ target_link_libraries(daemon
 add_dependencies(daemon version)
 set_property(TARGET daemon
   PROPERTY
-    OUTPUT_NAME "graftnoded")
+    OUTPUT_NAME "graftnoded.stable")
 install(TARGETS daemon DESTINATION bin)
--- graft-1.5.1.orig/src/simplewallet/CMakeLists.txt
+++ graft-1.5.1/src/simplewallet/CMakeLists.txt
@@ -59,5 +59,5 @@ add_dependencies(simplewallet
   version)
 set_property(TARGET simplewallet
   PROPERTY
-    OUTPUT_NAME "graft-wallet-cli")
+    OUTPUT_NAME "graft-wallet-cli.stable")
 install(TARGETS simplewallet DESTINATION bin)
--- graft-1.5.1.orig/src/wallet/CMakeLists.txt
+++ graft-1.5.1/src/wallet/CMakeLists.txt
@@ -119,7 +119,7 @@ if (NOT BUILD_GUI_DEPS)
   add_dependencies(wallet_rpc_server version)
   set_property(TARGET wallet_rpc_server
     PROPERTY
-      OUTPUT_NAME "graft-wallet-rpc")
+      OUTPUT_NAME "graft-wallet-rpc.stable")
   install(TARGETS wallet_rpc_server DESTINATION bin)
 endif()
 
